src/bundle/ui-dev/src/modules/universal-discovery/components/selected-items/selected.items.panel.item.js

```diff
@@ -0,0 +1,82 @@
+import React, { useContext, useEffect, useMemo, useRef } from 'react';
+
+import {
+    parse as parseTooltip,
+    hideAll as hideAllTooltips,
+} from '@ibexa-admin-ui/src/bundle/Resources/public/js/scripts/helpers/tooltips.helper';
+import { getAdminUiConfig, getTranslator } from '@ibexa-admin-ui/src/bundle/Resources/public/js/scripts/helpers/context.helper';
+
+import Icon from '../../../common/icon/icon';
+import Thumbnail from '../../../common/thumbnail/thumbnail';
+
+import { SelectedItemsContext } from '../../universal.discovery.module';
+
+import { REMOVE_SELECTED_ITEMS } from '../../hooks/useSelectedItemsReducer';
+
+const SelectedItemsPanelItem = ({ item, thumbnailData, name, description }) => {
+    const adminUiConfig = getAdminUiConfig();
+    const Translator = getTranslator();
+    const refSelectedLocationsItem = useRef(null);
+    const { dispatchSelectedItemsAction } = useContext(SelectedItemsContext);
+    const removeItemLabel = Translator.trans(
+        /*@Desc("Clear selection")*/ 'selected_items_panel.item.remove_item',
+        {},
+        'ibexa_universal_discovery_widget',
+    );
+    const removeFromSelection = () => {
+        hideAllTooltips(refSelectedLocationsItem.current);
+        dispatchSelectedItemsAction({ type: REMOVE_SELECTED_ITEMS, ids: [{ id: item.id, type: item.type }] });
+    };
+    const sortedActions = useMemo(() => {
+        const { universalSelectItemActions } = adminUiConfig.universalDiscoveryWidget;
+        const actions = universalSelectItemActions ? [...universalSelectItemActions] : [];
+
+        return actions.sort((actionA, actionB) => {
+            return actionB.priority - actionA.priority;
+        });
+    }, []);
+
+    useEffect(() => {
+        parseTooltip(refSelectedLocationsItem.current);
```

Comment:
I know that's totally not place for it, but just a hunch I want to mention - shouldn't we put `hideAllTooltips` in return function? As probably we should make sure we hide all tooltips when component is destroyed (marginally low chance for such case, as such component would need to cease to exist independently from user input, as user needs to hover cursor over tooltipable element)

Reply:
We should put it in the _ref_ function, I copied this code from another place and forgot to refactor it. :)